extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -28,6 +27,7 @@ export class ResourceTypePickerDialog extends DialogBase {
 	private _agreementContainer!: azdata.DivContainer;
 	private _agreementCheckboxChecked: boolean = false;
 	private _installToolButton: azdata.window.Button;
+	private _tools: ITool[] = [];
```

Comment:
No need to initialize when you're going to be setting it at the constructor.

Reply:
not initialized in the constructor.

Reply:
![image](https://user-images.githubusercontent.com/1465785/66677517-a866e380-ec1e-11e9-91a1-f3cbfd3c3f92.png)